cmake/external_rules.cmake

```diff
@@ -117,6 +117,10 @@ function(download_external_sources)
       file(INSTALL "${PROJECT_SOURCE_DIR}/cmake/external/leveldb.cmake"
            DESTINATION "${PROJECT_BINARY_DIR}/external/src/firestore/cmake/external")
     endif()
+
```

Comment:
Since we are now patching a couple of different things (this, boringssl, leveldb's cmake file), if there's ever even *one* more I'd sure love to see this changed to a generic patcher script, maybe with a json config file or something. :)